docs/components/table-action-button/README.md

```diff
@@ -0,0 +1,20 @@
+
+The component ```ftw-table-action-button``` was created to ensure all action-buttons in tables look similar.
+
+## Usage
+
+```vue
+<template>
+  <ftw-serverside-collection>
+    <template #actions="{ items }">
+      <ftw-table-action-button />
+    </template>
+  </ftw-serverside-collection>
+</template>
+```
+
+##Slots
+
+| Name            | Description                                                                            |
+|-----------------|----------------------------------------------------------------------------------------|
+| ``default``     | Slot to set label of the button |
```

Comment:
I'd reference the vuetify documentation here, since the button behaves exactly as documented under: https://vuetifyjs.com/en/api/v-btn/